lib/fluent/plugin/exception_detector.rb

```diff
@@ -76,6 +76,7 @@ def self.supported
 
     GO_RULES = [
       rule(:start_state, /\bpanic: /, :go_after_panic),
+      rule(:start_state, /http: panic /, :go_goroutine),
```

Comment:
Looking at https://golang.org/src/net/http/server.go, this could probably be:

```
      rule(:start_state, /http: panic serving /, :go_goroutine),
```

Reply:
That seems to be what is in the unit test, so all good.